src/components/middlepart/middlepart.module.scss

```diff
@@ -0,0 +1,126 @@
+@import "../../styles/variable";
+@import "../../styles/mixins";
+
+.outside {
+  background-color: $seperationColor;
+  padding: 5px 0;
+}
+.generalContainer {
+  @include centerStuff;
+  background-color: $defaultColor;
+  color: $textColor;
+  padding: 25px 60px;
+}
+.contentContainer {
+  width: 40%;
+  padding: 0 65px;
+  .title {
+    font-size: 3.5rem;
+  }
+  .subtitle {
+    font-size: 1.6rem;
+    font-weight: 400;
+  }
+}
+.videoContainer {
+  width: 48%;
+  height: 100%;
+  flex: 0 1 auto;
+  box-sizing: border-box;
+}
+.animationContainer {
+  position: relative;
+  overflow: hidden;
+}
+.tv {
+  max-width: 100%;
+  height: auto;
+  border: 0;
+  z-index: 100;
+  position: relative;
+  z-index: 2;
+}
+.videoDiv {
+  width: 100%;
+  height: 100%;
+  max-width: 73%;
+  max-height: 54%;
+  position: absolute;
+  top: 46%;
+  left: 49%;
+  transform: translate(-50%, -50%);
+
+  .video {
+    object-fit: contain;
+    display: inline-block;
+    vertical-align: baseline;
+  }
+}
+
+.videoDiv2 {
+  max-width: 63%;
+  max-height: 47%;
+  position: absolute;
+  top: 33%;
+  left: 46%;
+  transform: translate(-50%, -50%);
+  .video {
+    object-fit: contain;
+    width: 110%;
+    height: 100%;
+  }
+}
+
+// Downloading div
+.downloading {
+  padding: 0.5em 0.75em;
+  position: absolute;
+  left: 21%;
+  bottom: 12%;
+  z-index: 3;
+  margin: 0 auto;
+  background: $defaultColor;
+  display: flex;
+  align-items: center;
+  width: 50%;
+  min-width: 15 em;
+  padding: 0.25 em 0.65 em;
```

Comment:
is this should be `  padding: 0.25em 0.65em;` why there is space between number and it's unit